test/controllers/items_controller_test.rb

```diff
@@ -1,24 +0,0 @@
-require 'test_helper'
```

Comment:
The `test` directory can be completely removed.

The rails generators will continue to recreate the `test` directory and artifacts below it however.

See this SO answer for how to stop this from happening: http://stackoverflow.com/a/30379845/67834